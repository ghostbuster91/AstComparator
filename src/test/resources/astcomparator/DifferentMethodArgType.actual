class ActualClassName {

    void second(Integer a){
    
    }
}