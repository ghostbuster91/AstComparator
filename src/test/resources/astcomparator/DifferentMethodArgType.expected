class ActualClassName {

    void second(String a){
    
    }
}